multi-int-store/src/main/java/com/connexta/multiintstore/repositories/SearchRepository.java

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright (c) 2019 Connexta, LLC
+ *
+ * Released under the GNU Lesser General Public License version 3; see
+ * https://www.gnu.org/licenses/lgpl-3.0.html
+ */
+package com.connexta.multiintstore.repositories;
+
+import com.connexta.multiintstore.models.SearchDocument;
+import java.util.List;
+import org.springframework.data.solr.repository.SolrCrudRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface SearchRepository extends SolrCrudRepository<SearchDocument, String> {
```

Comment:
For consistency, wouldn't this be better named "SearchDocumentRepository"?

Reply:
Of the examples I've been looking at on the web, one put the word Document before Repository:
  https://www.petrikainulainen.net/programming/solr/spring-data-solr-tutorial-crud-almost/

and four did not:
  https://www.baeldung.com/spring-data-solr
  https://github.com/christophstrobl/spring-data-solr-showcase/tree/master/src/main/java/org/springframework/data/solr/showcase/product
  https://dzone.com/articles/crud-application-using-spring-data-solr-and-spring
  https://github.com/spring-projects/spring-data-solr-examples/tree/master/spring-solr-repository-example/src/main/java/org/springframework/data/solr/example/repository

I like the shorter "SearchRepository", but I would be OK with "SearchDocumentRepository"